tests/EdFi.Ods.Api.IntegrationTests/AuthorizationViewHelper.cs

```diff
@@ -0,0 +1,76 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Data;
+using Shouldly;
+using System.Linq;
+namespace EdFi.Ods.Api.IntegrationTests
+{
+    public static class AuthorizationViewHelper
+    {
+        public static List<EdFiTuple> GetAllInAuthorizationView(IDbConnection connection, string viewName, PersonType personType, string target = null)
```

Comment:
This will probably be consumed by QueryAuthorizationByViewName once the Student/Parent version of this lands.